src/modules/rtpengine/rtpengine.c

```diff
@@ -1675,6 +1677,19 @@ static int build_rtpp_socks() {
 				LM_WARN("Failed enable set MTU discovery socket option\n");
 #endif
 
+			if((0 <= control_cmd_tos) && (control_cmd_tos < 256)) {
+				if (pnode->rn_umode == 6) {
+					setsockopt(rtpp_socks[pnode->idx], IPPROTO_IPV6,
+							IPV6_TCLASS, &control_cmd_tos,
+							sizeof(control_cmd_tos));
+
+				} else {
+					setsockopt(rtpp_socks[pnode->idx], IPPROTO_IP,
+							IP_TOS, &control_cmd_tos,
+							sizeof(control_cmd_tos));
```

Comment:
If memory serves right, the argument to IP_TOS should be a single byte `unsigned char`.

Reply:
I have updated the code based on your review, if anything needs to be changed let me know please. Thanks for the review.